mit-bitcoin-expo/mit-bitcoin-expo-2022/lightning-panel.md

```diff
@@ -100,14 +100,14 @@ Maybe Rene?
 
 Rene Pickhardt: 00:06:38
 
-I don't know the exact number right now, but I think we have something like 70,000 channels that we see on Gossip.
+I don't know the exact number right now, but I think we have something like 70,000 channels that we see on `Gossip`.
```

Comment:
backticks also not needed here. As I said above, please remove them from wherever you have used them. 
We will try to come up with better guidance regarding the usage of backticks.